lib/serializables/log-event.js

```diff
@@ -32,7 +32,18 @@ function LogEventFactory (
     pretty,
     console
 ) {
+    //default turn off the colorful output to achieve tidy and clean log file.
+    //however user can still enable it via command line argument.
+    var colorEnable = false;
+    var argv = process.argv;
```

Comment:
It seems very unintuitive (and likely to be lost in the future) to know that we're inspecting arguments to the nodes process within this module. While this would function, command line options would be much better suited to being captured in a single location - I recommend in the top-level index.js - and relevant options then set internally or passed into this module to enable/disable color.